src/tests/unit/jest.config.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     roots: [currentDir],
     collectCoverage: true,
     collectCoverageFrom: ['<rootDir>/src/**/*.{ts,tsx}', '!<rootDir>/src/tests/**/*', '!<rootDir>/src/**/*.d.ts'],
-    coverageReporters: ['json', 'lcov', 'text', 'cobertura'],
+    coverageReporters: ['json', 'lcov', 'text', 'cobertura', 'html'],
```

Comment:
can we only add the reporters that we need?

Reply:
I thought about this, but I am unsure of the ones that we don't need & since this folder is ignored in the git, I dont know if that really makes a difference